src/file.js

```diff
@@ -24,22 +24,8 @@ Sk.builtin.file = function (name, mode, buffering) {
     } else if (this.name === "/dev/stderr") {
         this.fileno = 2;
     } else {
-        if (Sk.inBrowser) {  // todo:  Maybe provide a replaceable function for non-import files
```

Comment:
I **can** have my custom read function do both, but it seems risky to combine to different "virtual file systems" into one read.  Which should take precedence?